editor-desktop/build.gradle

```diff
@@ -8,7 +23,10 @@ dependencies {
     implementation libs.logback.classic
 
     implementation variantOf(libs.tiny.gizmo) { classifier("natives-desktop") }
+    implementation libs.javapoet
 
+    compileOnly 'org.projectlombok:lombok:1.18.22'
```

Comment:
Don't hard code dependenices like these. It should be done using libs as every other dep is done. idk why you even have lombok here

Reply:
fixed